Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py

```diff
@@ -2571,7 +2572,24 @@ def create_get_group_members_request(group_id=-1, limit=100):
     response = http_request('POST', api_endpoint, payload)
     if isinstance(response, dict) and response.get('fail'):
         raise Exception(json.dumps(response.get('fail', [{}])[0].get('errors')))
-    return response
+
+    members.update(response)
+
+    while int(limit) > len(members['data'][0]['groupMembers']) and response['meta']['pagination'].get('next'):
```

Comment:
lets add an argument all_results which can get true/false- by default will be set to false
and only if it is set to true we will do the while section